service/matching/matchingEngine.go

```diff
@@ -296,6 +296,21 @@ func (e *matchingEngineImpl) AddDecisionTask(
 		return false, err
 	}
 
+	// get the domainName
+	domainName, err := e.domainCache.GetDomainName(domainID)
+	if err != nil {
+		return false, err
+	}
+
+	// Only emit traffic metrics if the tasklist is not sticky and is not forwarded
+	if taskListKind != nil && *taskListKind != types.TaskListKindSticky && request.ForwardedFrom == "" {
+		e.metricsClient.Scope(metrics.MatchingAddTaskScope).Tagged(metrics.DomainTag(domainName),
+			metrics.TaskListTag(taskListName), metrics.TaskListTypeTag("activity_task")).IncCounter(metrics.CadenceTasklistRequests)
```

Comment:
Tasklist type tag was changed from "decision_task" to "activity_task" in your latest commit, is it by accident or you combined the types?

Reply:
It was an error, ty for spotting